include/gatekeeper_net.h

```diff
@@ -443,11 +443,67 @@ int convert_ip_to_str(const struct ipaddr *ip_addr, char *res, int n);
 int ethertype_filter_add(struct gatekeeper_if *iface, uint16_t ether_type,
 	uint16_t queue_id);
 
-int ntuple_filter_add(struct gatekeeper_if *iface, uint32_t dst_ip,
+/*
+ * Add a filter for IPv4 packets based on the destination IP address,
+ * source and destination ports, and protocol.
+ *
+ * @dst_ip: destination IP address to match against
+ * @src_port: L4 source port to match against
+ * @src_port_mask: mask for @src_port; set to 0xFFFF to match
+ *     against @src_port, or set to 0 to not match against @src_port
+ * @dst_port: L4 destination port to match against
```

Comment:
We should document that `@src_port` and `@dst_port` should be in big endian.

This comment also applies to `ipv6_ntuple_or_acl_add()` below.